src/main/java/org/gridsuite/modification/server/repositories/equipmentmodification/BatteryModificationRepository.java

```diff
@@ -0,0 +1,26 @@
+/*
+  Copyright (c) 2024, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.repositories.equipmentmodification;
+
+import org.gridsuite.modification.server.entities.equipment.modification.BatteryModificationEntity;
+import org.springframework.data.jpa.repository.EntityGraph;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+@Repository
+public interface BatteryModificationRepository extends JpaRepository<BatteryModificationEntity, UUID>, EagerNetworkModificationRepository<BatteryModificationEntity> {
+
+    @Override
+    @EntityGraph(attributePaths = {"reactiveCapabilityCurvePoints", "properties"}, type = EntityGraph.EntityGraphType.LOAD)
```

Comment:
I thought it was not possible to fetch two distinct collections from the same entity in JPA, see below exception
"org.hibernate.loader.MultipleBagFetchException: cannot simultaneously fetch multiple bags"